packages/react-form-state/src/validators.ts

```diff
@@ -48,9 +48,27 @@ export function validateNested<Input extends Object, Fields>(
   // eslint-disable-next-line consistent-return
   return (input: Input, fields: Fields) => {
     const errors = mapObject<Input, any>(input, (value, field) => {
-      return (
-        validatorDictionary[field] && validatorDictionary[field](value, fields)
-      );
+      if (validatorDictionary[field]) {
```

Comment:
doing an intermediary const and an early return instead of wrapping the whole function in an if is more idiomatic to the repo's code style.

```
const validate = validatorDictionary[field];

if (validate == null) {
  return null;
}

if (typeof validate === 'function') {
```